CHANGELOG.md

```diff
@@ -1 +1,38 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 17.11.0-ce (2017-11-DD)
+
+IMPORTANT: Docker CE 17.11 is the first Docker release based on
+[containerd 1.0 beta](https://github.com/containerd/containerd/releases/tag/v1.0.0-beta.2).
+Docker CE 17.11 and later won't recognize containers started with
+previous Docker versions. If using
+[Live Restore](https://docs.docker.com/engine/admin/live-restore/#enable-the-live-restore-option),
+you must stop all containers before upgrading to Docker CE 17.11.
+If you don't, any containers started by Docker versions that predate
+17.11 won't be recognized by Docker after the upgrade and will keep
+running, un-managed, on the system.
```

Comment:
keep running on the system, but Docker will not be able to see or manage them.

(Do we have a way for them to stop the containers if they do screw this up? What's the impact on swarms? Can we just make Docker detect this situation and stop those  containers itself? What if they had restart policies in place?)

Reply:
it won't impact swarm as swarm is incompatible with `live restore`, it's only used for "classic containers" not services.

@crosbymichael @friism @andrewhsu should we still package container 0.2 ctr client to stop those containers after an upgrade ? ( assume today an update would update ctr 0.2 to 1.0 so it won't be able to see the old container?)